CHANGELOG.md

```diff
@@ -1,16 +1,20 @@
 # Changelog
+All notable changes to this project will be documented in this file.
 
-## Draft
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 5.7.1 (07-09-2021)
+## Unreleased
```

Comment:
'Draft' should be used here as we are updating changelog with help of CLI

Reply:
reverted that commit now @BC-tymurbiedukhin

ref: https://github.com/bigcommerce/stencil-cli/blob/d9c0f3d3febdf4b51ae45a32fb534d726e3729f3/lib/release/release.js#L96